.eslintrc.js

```diff
@@ -26,7 +25,10 @@ module.exports = {
       }
     ],
     '@typescript-eslint/camelcase': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off'
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    'react/prop-types': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
```

Comment:
Our preferred style has to keep this in and disable when necessary:

```js
// eslint-disable-next-line @typescript-eslint/no-empty-function
```

Reply:
fixed